rust/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+### 0.7.2
+* Thread and memory safety fixes in mediasoup-sys
```

Comment:
```suggestion
### 0.7.2

* Thread and memory safety fixes in mediasoup-sys
```

Reply:
What happened to 0.7.1?

Reply:
Nothing, quite literally. There was just a version bump due to documentation build errors.